README.md

```diff
@@ -153,6 +153,21 @@ In the docker container, all the required prerequisites are already installed, s
 libs](#building-migraphx-source-and-install-libs)
 section to build MIGraphX source.
 
+### Using MIGraphX Python Module
+To use MIGraphX's Python module, we need to create MIGraphX .deb package and install it in order to import migraphx module successfuly to Python environment.
```

Comment:
This should also work with the packages from repo.radeon.com as well. For users that dont want to install a package you should let them know that they can set `PYTHONPATH` to their migraphx installation.

Reply:
I forgot to reply here after I have made the change :) 
I have added PYTHONPATH, let me know if it looks okay.

Reply:
@pfultz2 just a reminder